### DISCUSSION TITLE
Unbind Ctrl-k so it can do its thing again (kill line backwards)

### DISCUSSION THREAD

**Discussion by metaturso** [ajeetdsouza/zoxide#832] [Q&A] [answered]
When using Zoxide's Backward Search integration with Bash most [Emacs-like Readline shortcuts](https://readline.kablamo.org/emacs.html) work as intended except for `Ctrl-k` (kill line backwards.)

Right now `Ctrl-k` moves the selection up. Is it possible to unbind `Ctrl-k` and let the shell do its thing?

**Comment by ajeetdsouza** [comment, marked as answer]
zoxide does not bind ctrl-k.

**Reply by metaturso** [reply]
I'm referring to this new interactive frontend to Backward Search `Ctrl-r`:

![zwut](https://github.com/ajeetdsouza/zoxide/assets/30650/af6aa202-b2b2-4095-ade8-e0e6ec31e15d)

I figured it was part of either `z` or `zoxide`? Apparently not, as you say, also because it still comes up when I comment out `zoxide init bash`.

You seem to know a lot of what is or it isn't, would you mind pointing me in the right direction? I appreciate your help.

**Reply by metaturso** [reply]
Turns out that's `fzf`! I'll check their documentation and direct my questions there. Thanks again.
